config.py

```diff
@@ -1,3 +1,6 @@
+# Set where you would like the photo files to be stored - "s3" or "sftp"
+SOURCE_CONNECTION = "s3"
```

Comment:
This is a new configuration variable that tells the program where to save the image data.